crates/bevy_render/src/render_phase/draw_state.rs

```diff
@@ -10,7 +10,9 @@ use bevy_utils::tracing::trace;
 use std::ops::Range;
 use wgpu::{IndexFormat, RenderPass};
 
-/// Tracks the current [`TrackedRenderPass`] state to ensure draw calls are valid.
+/// Tracks the state of a [`TrackedRenderPass`].
+///
+/// This is used to skip redundant operations on the [`TrackedRenderPass`] (e.g. setting an already set pipeline, binding an already bound bind group).
```

Comment:
```suggestion
/// This is used to skip redundant operations on the [`TrackedRenderPass`] (e.g. setting an already 
/// set pipeline, binding an already bound bind group). These operations can otherwise be fairly
/// costly due to IO to the GPU, so deduplicating these calls results in a speedup.
```

Reply:
I like this addition. Added 👍.